tests/unit/test_client.py

```diff
@@ -121,7 +129,6 @@ def test_constructor_defaults(self):
         self.assertIs(client._client_info, _CLIENT_INFO)
         self.assertIsNone(client._channel)
         self.assertIsNone(client._emulator_host)
-        self.assertIsNone(client._emulator_channel)
```

Comment:
what is this equal to now? Can we add an equals assertion?

Reply:
It is a function now.